crypto/fipsmodule/modes/internal.h

```diff
@@ -280,7 +280,8 @@ void gcm_gmult_ssse3(uint64_t Xi[2], const u128 Htable[16]);
 void gcm_ghash_ssse3(uint64_t Xi[2], const u128 Htable[16], const uint8_t *in,
                      size_t len);
 
-#if defined(OPENSSL_X86_64) && !defined(MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX)
+#if defined(OPENSSL_X86_64) && !defined(MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX) \
+  && !defined(MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX)
```

Comment:
```suggestion
  && !defined(MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX512)
```

Reply:
Thank you, I had corrected it on my end but forgot to save it :)